build/container/Makefile

```diff
@@ -1,12 +1,31 @@
 .DEFAULT_GOAL := build/base
 
-INFRASTRUCTURE_WORKSPACE       ?= workspace
-PROJECT_WORKSPACE              ?= $(CURDIR)/$(INFRASTRUCTURE_WORKSPACE)
-
-TARGET_DIR						= $(PROJECT_WORKSPACE)/target
-NRI_INTEGRATIONS_FILE			= $(CURDIR)/../embed/integrations.version
-NRI_PKG_DIR						?= ${PROJECT_WORKSPACE}/${BASE_OHIS}
-AGENT_BUILD_TARGET_DIR			?= $(CURDIR)/../../target/bin/linux_amd64
+INFRASTRUCTURE_WORKSPACE	?= workspace
+PROJECT_WORKSPACE		?= $(CURDIR)/$(INFRASTRUCTURE_WORKSPACE)
+
+TARGET_DIR			?= $(PROJECT_WORKSPACE)/target/$(GOOS)_$(DOCKER_ARCH)
+NRI_INTEGRATIONS_FILE		= $(CURDIR)/../embed/integrations.version
+NRI_PKG_DIR			?= ${PROJECT_WORKSPACE}/${BASE_OHIS}
+AGENT_BUILD_TARGET_DIR		?= $(CURDIR)/../../dist
+GOOS				?= linux
+DOCKER_ARCH				?= amd64
+DOCKER_BUILDX_CMD		?=
+DOCKER_BUILDX_ARGS		?= --platform $(GOOS)/$(DOCKER_ARCH)
```

Comment:
Not specifying `--load` will cause the image to not be sent back from the buildx container to the host's docker daemon, at least not on my local machine. The image won't be found later when issuing `docker images`, or trying to `docker push` it.

Does this work differently in the GHA environment, or is it handled elsewhere?

Reply:
At the moment this was just to get the containers to build. Guess this would have come up when I came to try and publish. I think by default it would be good to push the image to the local docker daemon. Can't see this being an issue.